src/importexport/mei/internal/meiexporter.cpp

```diff
@@ -132,6 +132,7 @@ bool MeiExporter::write(QIODevice& destinationDevice)
         out << String::fromStdString(strStream.str());
     }
     catch (char* str) {
+        (void)str;
```

Comment:
```suggestion
        UNUSED(str);
```

alternatively we could probably remove the variable name
```cpp
catch (char*) {
    return false;
}
```
I don't know if some compilers would complain about that

Reply:
Yes, UNUSED is better, I thought I tried it and saw it not working. May need new glasses...

Reply:
As per https://en.cppreference.com/w/cpp/language/try_catch `catch (char*)` is not permitted, unless I missread it.
Anyway, why taking chances ;-)